aci_interface_policies.tf

```diff
@@ -200,35 +201,37 @@ locals {
+  policy_group_type          = each.value.policy_group_type
+  policy_group               = each.value.policy_group
+  breakout                   = each.value.breakout
+  fex_id                     = each.value.fex_id
+  description                = each.value.description
+  shutdown                   = each.value.shutdown
+  role                       = each.value.role
+  port_channel_member_policy = try("${each.value.port_channel_member_policy}${local.defaults.apic.access_policies.interface_policies.port_channel_member_policies.name_suffix}", "")
 }
 
 locals {
   new_leaf_subinterface_configuration = flatten([
     for node in local.nodes : [
       for interface in try(node.interfaces, []) : [
         for subinterface in try(interface.sub_ports, []) : {
-          key               = format("%s/%s/%s/%s", node.id, try(interface.module, local.defaults.apic.interface_policies.nodes.interfaces.module), interface.port, subinterface.port)
-          node_id           = node.id
-          module            = try(interface.module, local.defaults.apic.interface_policies.nodes.interfaces.module)
-          port              = interface.port
-          sub_port          = subinterface.port
-          policy_group_type = try([for pg in local.access_policies.leaf_interface_policy_groups : pg.type if pg.name == subinterface.policy_group][0], "access")
-          policy_group      = try("${subinterface.policy_group}${local.defaults.apic.access_policies.leaf_interface_policy_groups.name_suffix}", "system-ports-default")
-          fex_id            = try(subinterface.fex_id, "unspecified")
-          description       = try(subinterface.description, "")
-          shutdown          = try(subinterface.shutdown, false)
-          role              = node.role
+          key                        = format("%s/%s/%s/%s", node.id, try(interface.module, local.defaults.apic.interface_policies.nodes.interfaces.module), interface.port, subinterface.port)
+          node_id                    = node.id
+          module                     = try(interface.module, local.defaults.apic.interface_policies.nodes.interfaces.module)
+          port                       = interface.port
+          sub_port                   = subinterface.port
+          policy_group_type          = try([for pg in local.access_policies.leaf_interface_policy_groups : pg.type if pg.name == subinterface.policy_group][0], "access")
+          policy_group               = try("${subinterface.policy_group}${local.defaults.apic.access_policies.leaf_interface_policy_groups.name_suffix}", "system-ports-default")
+          fex_id                     = try(subinterface.fex_id, "unspecified")
+          description                = try(subinterface.description, "")
+          shutdown                   = try(subinterface.shutdown, false)
+          role                       = node.role
+          port_channel_member_policy = try(subinterface.port_channel_member_policy, "")
```

Comment:
Is it supported on sub-interfaces?

Reply:
well, its not supported on 5.2 but its supported starting 6.0